app/Enums/AccessibilityProjects.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Enums;
+
+use App\Enums\Traits\EnumHelper;
+
+enum AccessibilityProjects: string
+{
+    use EnumHelper;
+
+    case PROJECTMONITORING_DASHBOARD = "project monitoring:dashboard";
+    case PROJECTMONITORING_PROJECT = "project monitoring:projects";
+    case PROJECTMONITORING_MARKETING = "project monitoring:marketing";
```

Comment:
wala na dapat ning marketing lang. kay ma overwrite ni sa mga marketing_...
unless for group checking lang.  
addi lang og _ ang value then _GROUP sa var name